config.bal

```diff
@@ -1,5 +1,5 @@
-configurable string dbHost = "dbHost";
+configurable string dbHost = "medicinesforlk-mysql";
```

Comment:
this file commit is not required

Reply:
@AmilaJ  yes, I'm using it for the MySQL Docker image I'm creating.